src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenADependencyContextBuilder.cs

```diff
@@ -52,6 +52,7 @@ public class GivenADependencyContextBuilder
                 ReferenceInfo.CreateDirectReferenceInfos(
                     referencePaths ?? new ITaskItem[] { },
                     referenceSatellitePaths ?? new ITaskItem[] { },
+                    projectReferenceExistedInProjectContext: false,
```

Comment:
Naming suggestion: `projectContextHasProjectReferences` (here and wherever else this name is used.

Elsewhere it was confusing me that this seemed to be referring to a single project reference rather than the general set of all project references.

Reply:
renamed